server/tests/routes/group.test.js

```diff
@@ -329,4 +329,95 @@ describe('## Group API', () => {
       });
     });
   });
+
+  describe('# POST /api/v1/group/:groupName/request', () => {
+    let groupAdminUserAndToken;
+    let adminUserAndToken;
+    let userAndToken;
+    let group;
+
+    beforeEach((done) => {
+      const groupAdminUserBuild = buildAndAuth('user', fixtures.user())
+        .then(createdUserAndToken => (groupAdminUserAndToken = createdUserAndToken));
+      const adminUserBuild = buildAndAuth('user', fixtures.user({ isSysAdmin: true }))
+        .then(createdUserAndToken => (adminUserAndToken = createdUserAndToken));
+      const userBuild = buildAndAuth('user', fixtures.user())
+        .then(createdUserAndToken => (userAndToken = createdUserAndToken));
+
+      Promise.all([groupAdminUserBuild, adminUserBuild, userBuild])
+        .then(values => build('group', fixtures.group({
+          requests: [values[2].user.id],
+          admins: [values[0].user.id]
+        })))
+        .then((g) => {
+          group = g;
+          done();
+        });
+    });
+
+    context('with group admin permissions', () => {
+      it('accepts a request', (done) => {
```

Comment:
The functionality between the service and the route isn't different, the route tests were made just to test the permissions on the route.